### ISSUE TITLE
[Reimplementation] After the SODA-A dataset is split, there is a lot of JSON data of annotations in the train. Do they need to be integrated?

### ISSUE THREAD

**Issue by 1339630594** [shaunyuan22/CFINet#38]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmdetection/issues) and [Discussions](https://github.com/open-mmlab/mmdetection/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (master)](https://github.com/open-mmlab/mmdetection) or [latest version (3.x)](https://github.com/open-mmlab/mmdetection/tree/dev-3.x).

### 💬 Describe the reimplementation questions

After the SODA-A dataset is split, there is a lot of JSON data of annotations in the train. Do they need to be integrated?

### Environment

pip install pytorch

**Comment by shaunyuan22** [comment]
nope, please refer to [here](https://github.com/shaunyuan22/SODA-mmrotate/tree/main/tools/data/sodaa) for data preparation of SODA-A dataset.
